tests/create_golden_values.py

```diff
@@ -0,0 +1,31 @@
+'''
+Generate a golden NPZ file from a dicom ZIP archive.
+'''
+import argparse
+
+import numpy as np
+
+from dicom_numpy.zip_archive import combined_series_from_zip
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-o', '--output', help='Output golden NPZ file', required=False)
+    parser.add_argument('input', help="Input DICOM zip archive")
+    return parser.parse_args()
+
+
+def generate_golden_values(input_zip, output_path='golden_values'):
```

Comment:
If the number of datasets for this grows and/or we are updating the golden values more frequently (both seem unlikely), then we should consider using [HDAT](https://github.com/innolitics/hdat) to produce the golden values.

I definitely think what you have is appropriate to start with, since it is simpler than using HDAT (and bc the golden values seem unlikely to change often).